### DISCUSSION TITLE
Are Playbook Modifications in Security Onion You Make Public?

### DISCUSSION THREAD

**Discussion by Ryan8bit** [Security-Onion-Solutions/securityonion#10440] [General]
Are the Playbook modifications in Security Onion that you make public?

**Comment by dougburks** [comment]
Are you looking for source code?

Here's the Docker image for Playbook:
https://github.com/Security-Onion-Solutions/securityonion-image/tree/master/so-playbook

Here's the Playbook plugin:
https://github.com/Security-Onion-Solutions/securityonion-playbook-plugin
